doc_admin/OH_software_compatibility_matrix.adoc

```diff
@@ -1,6 +1,6 @@
 [width="99%",cols="^16%,^14%,^14%,^14,^14%,^14%,^14%,^14%,^14%",options="header"]
 |===
-9+|*Open Hospital compatibility matrix*
+9+|*Open Hospital software compatibility matrix*
 
 .2+|*OH version* .2+|*MariaDB* / MySQL (community version) .2+|*JAVA* JRE version (*) .2+|O.S. .2+|O.S. arch .2+|JAVA/MariaDB arch 3+|Special Features
 |DICOM (*) |SMS (gsm) |Webcam
```

Comment:
Since 1.12.0 is using the more general term "Imaging" or "Image" should this be changed?